xapian-applications/omega/query.cc

```diff
@@ -1064,6 +1066,7 @@ T(fmt,		   0, 0, N, 0), // name of current format
 T(freq,		   1, 1, N, 0), // frequency of a term
 T(ge,		   2, 2, N, 0), // test >=
 T(gt,		   2, 2, N, 0), // test >
+T(hash,		   2, 2, N, 0), // hash a string using the specified hash function
 T(highlight,	   2, 4, N, 0), // html escape and highlight words from list
```

Comment:
It's because there's an extra column at the start of the patch (with `+` for added lines, `-` for removed lines and ` ` for context lines).  In the normal file, the `,` is column 7 and the tab takes us to column 8.  In the patch, the `,` is column 8 so that same tab takes us to column 16.

So don't worry about it.  It's not you.